src/Model/Scope/CompoundCondition.php

```diff
@@ -194,14 +194,16 @@ public function toWords(Model $model = null): string
      */
     public static function createAnd(...$conditions)
     {
-        return new static($conditions, self::AND);
+        // intentional use of self
+        return new self($conditions, self::AND);
     }
 
     /**
      * @return static
      */
     public static function createOr(...$conditions)
     {
-        return new static($conditions, self::OR);
+        // intentional use of self
+        return new self($conditions, self::OR);
```

Comment:
what did you solve? `Scope` was fixed to be using `parent::`, ie. `self`, but imagine this - you extend this method with some other custom class and what would you expect from `createOr`? I `new static`.